src/main/webapp/app/exercises/quiz/shared/questions/short-answer-question/short-answer-question.component.html

```diff
@@ -48,18 +48,22 @@ <h4 class="question-title-display">
                         value="{{ getSubmittedTextForSpotAsString(element) }}"
                         size="{{ getSubmittedTextSizeForSpot(element) }}"
                     />
-                    <div
-                        *ngIf="shortAnswerQuestionUtil.getSpot(shortAnswerQuestionUtil.getSpotNr(element), shortAnswerQuestion).invalid"
-                        [ngbTooltip]="'artemisApp.shortAnswerSpot.invalidSpot' | artemisTranslate"
-                    >
-                        <input
-                            class="short-answer-question-container__input"
-                            style="background: grey"
-                            disabled
-                            value="{{ getSubmittedTextForSpotAsString(element) }}"
-                            size="{{ getSubmittedTextSizeForSpot(element) }}"
-                        />
-                    </div>
+                </div>
+                <div
+                    *ngIf="
+                        shortAnswerQuestionUtil.isInputField(element) &&
+                        !showingSampleSolution &&
+                        shortAnswerQuestionUtil.getSpot(shortAnswerQuestionUtil.getSpotNr(element), shortAnswerQuestion).invalid
+                    "
+                    [ngbTooltip]="'artemisApp.shortAnswerSpot.invalidSpot' | artemisTranslate"
+                >
+                    <input
+                        class="short-answer-question-container__input"
+                        style="background: grey"
```

Comment:
Can we move this inline style to ``short-answer-question.component.scss`` as we're already refactoring it?